src/compiler/symb-proc.bas

```diff
@@ -2298,6 +2298,14 @@ function symbFindUopOvlProc _
 		if( *err_num <> FB_ERRMSG_OK ) then
 			errReport( *err_num, TRUE )
 		end if
+	else
+		'' check visibility
+		if( symbCheckAccess( proc ) = FALSE ) then
```

Comment:
shouldn't the same be added to `symbFindBopOvlProc()`?

Reply:
oops, yes.  same visibility check should be added to symbFindBopOvlProc().

And now I see the 4 blocks of duplicate code ... my achilles heel.  Even a macro would be better to capture that all 4 procedures are doing the exact same error check.